src/pages/registrering/useFnr.ts

```diff
@@ -0,0 +1,47 @@
+import { useCallback, useEffect, useState } from 'react';
+import { hentFnr } from '../../api/aktørregisterApi';
+import { Feilmelding } from '../før-du-begynner/useAktørId';
+import { hentGjeldendeIdent } from '../../api/aktørregisterUtils';
+
+const useFnr = (aktørId: string): string => {
+    const [fnr, setFnr] = useState<string>('');
+
+    const hentOgSettFnr = useCallback(async () => {
+        try {
+            const fnr = await hentOgSjekkFnr(aktørId);
+            setFnr(fnr);
+        } catch (ignored) {}
+    }, [aktørId]);
+
+    useEffect(() => {
+        hentOgSettFnr();
+    }, [hentOgSettFnr]);
+
+    return fnr;
+};
+
+const hentOgSjekkFnr = async (aktørId: string) => {
+    let respons;
+    try {
+        respons = await hentFnr(aktørId);
+    } catch (error) {
+        // Hva skal skje om dette ikke går?
+        throw Feilmelding.Serverfeil;
```

Comment:
Hvordan skal vi håndtere feil ved henting av fnr?
Må eventuelt ha en feilmelding en plass...

Her må man vel legge til noen ekstra sjekker i backend for "validering" av aktørId så man ikke oppretter kandidater med vilkårlige aktørIdeer som ikke samsvarer med et fnr.